config/tests/mod.rs

```diff
@@ -0,0 +1,239 @@
+//! Tests for parsing configuration files.
+//!
+//! Test config files are located in the `tests/support/config` subdirectory.
+
+mod files {
```

Comment:
Is this file not the same as `config/tests/config.rs`?

Reply:
Also, what's the benefit of wrapping these tests in the `files` mod? Could we not just do away with the outer `mod files` wrapper?

Reply:
> Is this file not the same as `config/tests/config.rs`?

Oh yeah, that was probably a mistake during the file moving. I have removed the duplicate.


> Also, what's the benefit of wrapping these tests in the `files` mod? Could we not just do away with the outer `mod files` wrapper?

Yes we can. The `mod files` was there in the original code. I have moved them out now. One disadvantage is that git now recognize the changes as a new file rather than a moved file.

Reply:
> Yes we can. The mod files was there in the original code. I have moved them out now. One disadvantage is that git now recognize the changes as a new file rather than a moved file.

That's fine, thanks!